go/apps/mixnet/mixnet_router/mixnet_router.go

```diff
@@ -68,16 +69,28 @@ var x509Identity pkix.Name = pkix.Name{
 
 func main() {
 	flag.Parse()
-	timeout, _ := time.ParseDuration("5s") // TODO(cjpatton) make this a command line parameter.
-	hp, err := mixnet.NewRouterContext(*configPath, *serverNetwork, *serverAddr, *batchSize,
+	timeout, err := time.ParseDuration(*timeoutDuration)
+	if err != nil {
+		glog.Errorf("router: failed to parse timeout duration: %s", err)
```

Comment:
If we fail to parse the duration, I think we should exit. Ditto for other glog.Error* calls below